### ISSUE TITLE
Author test case: aria-busy

### ISSUE THREAD

**Issue by cyns** [w3c/aria#1543]
aria-busy
If changes to a rendered widget would create a state where the widget is missing required owned elements during script execution, authors MUST set aria-busy to true on the widget during the update process.
